piet-gpu-types/src/scene.rs

```diff
@@ -1,7 +1,7 @@
 use piet_gpu_derive::piet_gpu;
 
 pub use self::scene::{
-    Clip, CubicSeg, Element, FillColor, LineSeg, QuadSeg, SetLineWidth, Transform,
+    Clip, CubicSeg, Element, FillColor, LineSeg, QuadSeg, SetFillMode, SetLineWidth, Transform,
```

Comment:
I'm not 100% sure I understand the motivation for moving this to the element monoid; it seems fairly straightforward to encode into the tag word where it would be readily available.